front_end/messages/en.json

```
@@ -623,6 +623,11 @@
   "questionWeight": "Question Weight",
   "questionWeightTooltip": "This question has a weight of <bold>{weight}%</bold>, providing {count, plural, =1 {<bold>{weightDiff}% fewer points</bold>} other {a <bold>{weightDiff}% point boost</bold>}} compared to regular Metaculus questions.",
   "includeBots": "Include Bots",
+  "excludeBots": "Exclude Bots",
```

Biome (2.5.3)

[error] The key excludeBots was already declared.

(lint/suspicious/noDuplicateObjectKeys)